modulemd/v2/modulemd-service-level.c

```diff
@@ -55,6 +55,30 @@ modulemd_service_level_new (const gchar *name)
 }
 
 
+gboolean
+modulemd_service_level_equals (ModulemdServiceLevel *self_1,
```

Comment:
You need to include a prototype for this function in the `modulemd-service-level.h` header file. Follow the examples of the other functions in there for what the GTKDoc comments should be (and mark it as `Since: 2.2`).